### ISSUE TITLE
Zed won't open filename-line-number pattern from `grep -Hn'

### ISSUE THREAD

**Issue by zacharysyoung** [zed-industries/zed#10897]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

It looks like Zed expects a space after `some/path:lineno` patterns ("pattern") in the terminal for them to be clickable:

```
some/path/myfile:1 other extraneous text
``` 

`grep -Hn` prints the pattern, a colon, then immediately prints that line's text:

```
some/path/myfile:1:other extraneous text
``` 

It'd be nice if Zed would allow those patterns to be clickable.

### Environment

Zed: v0.131.7 (Zed)
OS: macOS 14.4.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

```shell
% grep -Hn '// A Scanner' fixedwidth/scanner.go
```

produces the un-clickable result:

```none
fixedwidth/scanner.go:18:// A Scanner reads records from a fixed-width-encode file.
```

I can replace the final colon from grep's output (which happens to be followed directly by two slashes '//') with a space:

```shell
% grep -Hn '// A Scanner' fixedwidth/scanner.go | sed 's/:\/\// \/\//'
```

and now the result is clickable:

```none
fixedwidth/scanner.go:18 // A Scanner reads records from a fixed-width-encode file.
```

**Comment by osiewicz** [comment]
It should be fixed with tomorrow's Preview. See also #10688

**Comment by Moshyfawn** [comment]
I'm going to close this issue as it was resolved in #10688. Keep an eye out for tomorrow's release 👀

**Comment by zacharysyoung** [comment]
Thanks! I really tried to find an existing issue, promise :) Just didn't know what to search for.
